src/installation/install_epinio_manual.md

```diff
@@ -0,0 +1,120 @@
+# Install Epinio (the manual way)
+
+As described [in the top level document](src/installation/installation.md#installation-methods), this method is intended for production level setups where operators want to have full control of the process.
+
+## Prerequisites
+
+Epinio needs a number of external components to be running on your cluster in order to work. You may already have those deployed, otherwise follow the instructions here to deploy them.
+
+Important: Some of the namespaces of the components are hardcoded in the Epinio
+code and thus are important to be the same as described here. In the future this
+may be configurable on the Epinio Helm chart.
+
+### Linkerd
+
+- Optional
+
+Download the linkerd cli from here: https://github.com/linkerd/linkerd2/releases/tag/stable-2.10.2
+
+Install linkerd with:
+
+```
+$ kubectl create namespace linkerd
+$ kubectl apply -f assets/embedded-files/linkerd/rbac.yaml 
+$ linkerd install | kubectl apply -f - && linkerd check --wait 10m
+```
+
+### Traefik
+
+Install Traefik with:
+
+```
+$ kubectl create namespace traefik
+$ export LOAD_BALANCER_IP=$(LOAD_BALANCER_IP:-) # Set this to the IP of your load balancer if you know that
+$ helm install traefik --namespace traefik "https://helm.traefik.io/traefik/traefik-10.3.4.tgz" \
+		--set globalArguments='' \
+		--set-string deployment.podAnnotations.linkerd\\.io/inject=enabled \
+		--set-string ports.web.redirectTo=websecure \
+		--set-string ingressClass.enabled=true \
+		--set-string ingressClass.isDefaultClass=true \
+		--set-string service.spec.loadBalancerIP=$LOAD_BALANCER_IP
+```
+
+### Install Kubed
+
+```
+$ kubectl create namespace kubed 
+$ helm repo add appscode https://charts.appscode.com/stable/
+$ helm repo update
+$ helm install kubed --namespace kubed --version v0.12.0 appscode/kubed
+```
+
+### Install Cert Manager
+
+```
+$ kubectl create namespace cert-manager
+$ helm repo add jetstack https://charts.jetstack.io
+$ helm repo update
+$ helm install cert-manager --namespace cert-manager jetstack/cert-manager \
+		--set installCRDs=true \
+		--set extraArgs[0]=--enable-certificate-owner-ref=true
+```
+
+### Install Tekton
+
+```
+$ kubectl create namespace tekton
+$ kubectl label namespace tekton "linkerd.io/inject"="enabled"
+$ kubectl apply -f https://raw.githubusercontent.com/epinio/epinio/main/assets/embedded-files/tekton/pipeline-v0.28.0.yaml
```

Comment:
```suggestion
$ kubectl apply -f https://raw.githubusercontent.com/epinio/installer/main/assets/installer/pipeline-v0.28.0.yaml
```

Reply:
Hmm, our file has only one line different from the upstream one here: https://github.com/tektoncd/pipeline/releases/download/v0.28.0/release.yaml

maybe fetching the original file and `sed`ing the line would be better? On the other hand, we may introduce more changes in the future.